### ISSUE TITLE
Statistics graph to show plays by stream type (Transcode, Direct Play)

### ISSUE THREAD

**Issue by rcs1234567** [CyferShepard/Jellystat#211]
Would like a graph on the statistics page which shows the number of plays by stream type (Transcode or Direct Play). 

It seems Jellystat already has this data in the Activity tab client field (Transcode vs Direct) but does not provide a graph for it

Tautulli has a similar graph which I find useful 'Daily concurrent stream count by stream type

**Comment by zodac** [comment]
This would be quite handy. I wonder if it would be possible to drill into the information and check which media was direct play/transcode too.

**Comment by JamsRepos** [comment]
Yeah this would be great. As well as even just showing a new field type on the already existing activity feed too.

**Comment by CyferShepard** [comment]
Added stat in 1.1.1, still need to add a graph

**Comment by antt0n** [comment]
I would like to add that it would be interesting to differentiate between video and audio transcoding. Because currently, no difference can be made between these two types of transcoding, which however do not have absolutely the same impact in terms of the power required.

![image](https://github.com/user-attachments/assets/ba490302-2285-458e-b85a-9fbde283c14c)

![image](https://github.com/user-attachments/assets/47e1e3d9-0e68-4abf-bd9b-686ad571e9d3)
